rest/api.go

```diff
@@ -346,7 +348,19 @@ func (h *handler) handlePprofProfile() error {
 }
 
 func (h *handler) handlePprofBlock() error {
+	sec, err := strconv.ParseInt(h.rq.FormValue("seconds"), 10, 64)
+	if sec <= 0 || err != nil {
+		sec = 30
+	}
+	if h.server.blockRunning.IsTrue() {
+		return base.HTTPErrorf(http.StatusForbidden, "Can only run one block profile at a time")
+	}
+	h.server.blockRunning.Set(true)
```

Comment:
This doesn't actually prevent all races - two instances could check true, then both make it through to 358.  Could change this to an atomic int flag and use CompareAndSwap to address this.

Reply:
Done